test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -4053,9 +4053,36 @@ public async Task GetInstalledAndTransitivePackagesAsync_ProjectReferenceWithSin
 
             // Assert
             Assert.Empty(result.InstalledPackages); // No installed packages
-            // Foreign transitive packages in assets file don't have dependencies.
-            // Those are not considered transitive packages
-            Assert.Empty(result.TransitivePackages);
+            Assert.Equal(1, result.TransitivePackages.Count);
```

Comment:
In this test, there's a mention of PackageA in the test, but package A does not exist.